### DISCUSSION TITLE
Does Ionic Capacitor support android 5 and android 6?

### DISCUSSION THREAD

**Discussion by to-svarghese** [ionic-team/capacitor#4204] [Q&A]
<img width="727" alt="Screenshot_26" src="https://user-images.githubusercontent.com/41439478/107912864-9b1edf00-6f5f-11eb-8c65-e9f406d74be4.png">
<img width="312" alt="Screenshot_27" src="https://user-images.githubusercontent.com/41439478/107912868-9bb77580-6f5f-11eb-9e86-c2ab7695f898.png">

I have created a small camera APP with Ionic Capacitor. I am not able to run this APP on Android 5 o6 . I could run from android 7+. I just want to know whether Capacitor really support on android 5 and 6?

**Comment by jcesarmobile** [comment]
They are supported as long as the Android WebView version is 50 or later, but the simulators use an older version that can’t be updated, so you should test on real devices.

**Reply by to-svarghese** [reply]
ok.  Thank you for your information
